B827EBFFFEBF4BED.json

```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "B827EBFFFEBF4BED", 
+		"servers": [ 
+			{ 
+				"server_address": "ttn-router-asia-se", 
```

Comment:
That's not a valid server address. Please check the valid ones in here: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses